packages/integration-karma/test/mixed-shadow-mode/synthetic-behavior/index.spec.js

```diff
@@ -49,6 +57,46 @@ if (!process.env.NATIVE_SHADOW && !process.env.COMPAT) {
                 tagName: 'x-parent-light-child-reset',
                 nativeLeaf: false,
             },
+            {
+                Component: GrandparentAnyParentAnyChildAny,
+                tagName: 'x-grandparent-any-parent-any-child-any',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentAnyParentAnyChildReset,
+                tagName: 'x-grandparent-any-parent-any-child-reset',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentAnyParentResetChildAny,
+                tagName: 'x-grandparent-any-parent-reset-child-any',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentAnyParentResetChildReset,
+                tagName: 'x-grandparent-any-parent-reset-child-reset',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentResetParentAnyChildAny,
+                tagName: 'x-grandparent-reset-parent-any-child-any',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentResetParentAnyChildReset,
+                tagName: 'x-grandparent-reset-parent-any-child-reset',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentResetParentResetChildAny,
+                tagName: 'x-grandparent-reset-parent-reset-child-any',
+                nativeLeaf: true,
+            },
+            {
+                Component: GrandparentResetParentResetChildReset,
+                tagName: 'x-grandparent-reset-parent-reset-child-reset',
+                nativeLeaf: false,
+            },
```

Comment:
I guess we could add Light mode tests here too, but maybe that would really be overkill. 🙂